protocols/twitter/twitter_lib.c

```diff
@@ -861,6 +861,9 @@ static void twitter_http_stream(struct http_request *req)
 		}
 
 		imcb_error(ic, "Stream closed (%s)", req->status_string);
+		if (req->status_code == 401) {
+			imcb_error(ic, "If you keep getting this, try checking your system clock");
```

Comment:
At the risk of sounding pedantic.. shouldn't this rather be just "check your system clock" and have a trailing dot ?